src/core/operators/defer_c.c

```diff
@@ -6,7 +6,7 @@
 
 // Single bigarray not registered with the GC.
 CAMLprim value liquidsoap_alloc_managed_int16_ba(value _len) {
-  CAMLparam0();
+  CAMLparam1(_len);
```

Comment:
Integers don't need to be registered with the GC because they don't hold any allocated memory!